docs/source/release/v3.10.0/framework.rst

```diff
@@ -61,6 +61,7 @@ Python
       3.0
       3.0
 
+- Exposed more `SpectrumInfo` functionality to Python.
```

Comment:
We have not "officially" release `SpectrumInfo` to Python (it was not included in previous release notes and there is no documentation apart form very basic Doxygen), so this line does not make sense here.